client/dashboard/index.js

```diff
@@ -14,6 +21,59 @@ import ProfileWizard from './profile-wizard';
 import withSelect from 'wc-api/with-select';
 
 class Dashboard extends Component {
+	componentDidUpdate( prevProps ) {
+		const profileItems = get( this.props, 'profileItems', {} );
+		const prevProfileItems = get( prevProps, 'profileItems', {} );
+
+		if ( profileItems.completed && ! prevProfileItems.completed ) {
+			updateQueryString( {}, '/', {} );
+			this.redirectToCart();
+		}
+	}
+
+	getProductIds() {
+		const productIds = [];
+		const profileItems = get( this.props, 'profileItems', {} );
+
+		profileItems.product_types.forEach( product_type => {
+			if (
+				wcSettings.onboarding.productTypes[ product_type ] &&
```

Comment:
Instead of using `wcSettings` directly, this should be using `getSetting`. I.e. something like

`import { getSetting } from '@woocommerce/wc-admin-settings';`
`const onboarding = getSetting( 'onboarding', {} );`